packages/wallet/backend/src/incomingPayment/service.ts

```diff
@@ -129,4 +139,16 @@ export class IncomingPaymentService implements IIncomingPaymentService {
 
     return `${existingPaymentPointer.url}/incoming-payments/${response.paymentId}`
   }
+
+  private generateExpiryObject(expiry: number, unit: string): Duration {
+    switch (unit) {
+      case 's': {
```

Comment:
this function also supports 'm' and 'h' as well?

Reply:
Yes, this will also support `m(inutes)`, `h(ours)` and `d(ays)`.